### ISSUE TITLE
6.7.0 handling of KFX

### ISSUE THREAD

**Issue by net-buoy** [apprenticeharper/DeDRM_tools#1006]
The new release info states:
"This release DOES NOT handle Kindle for Mac/PC 1.25 and later."
Can we take that to mean that if 6.6.3 is presently working with  Kindle for Mac/PC 1.25 or later, that the release will not impact that current functionality?

**Comment by ElleKayEm** [comment]
Correct.  This is referring to handling the newer KFX format DRM which you will have disabled for Kindle for Mac 1.26 or later to work.  (Kindle for PC users still need to stick with version 1.24 or earlier whether or not they disable KFX downloads.)
